/camera_node:
  ros__parameters:
    general_camera_path: "/dev/v4l/by-path/platform-fd500000.pcie-pci-0000:01:00.0-usb-0:1.1:1.0-video-index0"
    field_camera_path: "/dev/v4l/by-path/platform-fd500000.pcie-pci-0000:01:00.0-usb-0:1.3:1.0-video-index0"
    publish_timer: 0.03 # 10 fps
    frame_width: 640
    frame_height: 480

/flask_node:
  ros__parameters:
    port: 8080
    host: "0.0.0.0"
    joystick_offset: 0.01 # meters

/urx_node:
  ros__parameters:
    ip: "192.168.2.65"
    popup_message: "Манипулятор захвачен RobotX\nManipulator occupied by RobotX"
    status_publish_rate: 0.5 # second
    gripper_start_pose: 0 # 0 - 255
    gripper_step: 10 # 0 - 255


/robot_control:
  ros__parameters:
    gripper:
      max: 100
      min: 0
    movel: # meters
      x_min: -0.22
      x_max: 0.22

      y_max: -0.150
      y_min: -0.5

      z_min: 0.17
      z_max: 0.35

      velocity: 0.15
      acceleration: 0.15


# -0.25 0.25
# -0.150 -0.5
# 0.01 0.2
